### PR TITLE
Wizard automation: check for response on "info needed" issue & close

### PR DESCRIPTION
**Pull request by merelcht**
## Description
Any ticket labelled with `support: need more info` will be closed automatically after 14 days if the author doesn't respond. The label is removed if the author does respond.

## Developer Certificate of Origin
We need all contributions to comply with the [Developer Certificate of Origin (DCO)](https://developercertificate.org/). All commits must be signed off by including a `Signed-off-by` line in the commit message. [See our wiki for guidance](https://github.com/kedro-org/kedro/wiki/Guidelines-for-contributing-developers/).

If your PR is blocked due to unsigned commits, then you must follow the instructions under "Rebase the branch" on the GitHub Checks page for your PR. This will retroactively add the sign-off to all unsigned commits and allow the DCO check to pass.

## Checklist

- [ ] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Signed off each commit with a [Developer Certificate of Origin (DCO)](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-the-commit-signoff-policy-for-your-repository)
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes
- [ ] Checked if this change will affect Kedro-Viz, and if so, communicated that with the Viz team

### CODE REVIEW

**Comment by rashidakanchwala** [comment]
I am confused if we should close them or not? Can we not automate the closing part, as I feel it requires some manual intervention.

**Comment by merelcht** [comment]
> I am confused if we should close them or not? Can we not automate the closing part, as I feel it requires some manual intervention.

This GH action will close the ticket automatically if there's not response from the author after 14 days.

**Comment by merelcht** [comment]
> That makes sense. What I meant was that I feel undecided about automatically closing the issue in 14 days. It’s not uncommon for Viz-related community issues to remain open longer, often requiring multiple follow-ups to get user input. However, if you and @astrojuanlu are happy with the automatic 14-day closure, I’m ok with it too.

We can definitely make it longer! Or set a different limit for each repo. We'll have to add each action to the repos separately anyway.

**Review by noklam — APPROVED** [review]

**Review by ElenaKhaustova — APPROVED** [review]

File: .github/workflows/no-response.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  noResponse:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: lee-dohm/no-response@v0.5.0
+        with:
+          token: ${{ github.token }}
+          responseRequiredLabel: "support: needs more info"
+          daysUntilClose: 14
+          closeComment: >-
+            This issue has been closed due to lack of information.
```
Comment (merelcht) [inline .github/workflows/no-response.yml:20]:
```suggestion
            This issue has been closed due to lack of information. Feel free to re-open this issue if you're facing a similar problem. Please provide as much information as possible so we can help resolve your issue. 
```

**Review by astrojuanlu — COMMENTED** [review]
We can always reopen if more information is provided, so I think 14 days is OK 👍🏼

**Comment by astrojuanlu** [comment]
On second thought, wondering if 14 days might be too aggressive and we could bump it to 28.

**Comment by merelcht** [comment]
Of course, I'll do that now!

**Pull request merged**